nbs/docs/tutorials/20_conformal_prediction.ipynb

```diff
@@ -0,0 +1,230 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Conformal Prediction\n",
+    "> Tutorial on how to train neuralforecast models and obtain prediction intervals using the conformal prediction methods"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Conformal prediction intervals use cross-validation on a point forecaster model to generate the intervals. This means that no prior probabilities are needed, and the output is well-calibrated. No additional training is needed, and the model is treated as a black box. The approach is compatible with any model.\n",
+    "\n",
+    "In this notebook, we demonstrate how to obtain intervals using the conformal prediction methods."
```

Comment:
"In this notebook, we demonstrate how to obtain prediction intervals using conformal prediction."